src/quoting-client.js

```diff
@@ -35,6 +36,7 @@ class QuotingClient {
       let query = {
         source_ledger: source,
         destination_ledger: destination,
+        destination_account: destinationAccount,
```

Comment:
I'm trying to understand where "destinationAccount" has an effect that is not already determined by "destinationLedger". I assume that the connector is not going to discriminate between different users on the same ledger. I understand that "destinationAccount" has to be filled in in the final payment object. Is the answer perhaps that the pathfinding process can't progress unless _some_ valid account is placed in the template, and which _specifc_ account doesn't actually matter?